src/utils/registration-helpers.ts

```diff
@@ -818,30 +843,40 @@ export const convertToRegistrationSearchItem = (registration: Registration) => {
       ? registration.entityDescription.reference.publicationContext.series
       : undefined;
 
+  const contributors =
+    registration.entityDescription?.contributors.map((contributor) => ({
+      affiliation: contributor.affiliations,
+      correspondingAuthor: contributor.correspondingAuthor,
+      identity: contributor.identity,
+      role: contributor.role.type,
+    })) ?? [];
+
   const registrationSearchItem: RegistrationSearchItem = {
+    type: registration.entityDescription?.reference?.publicationInstance.type ?? '',
     id: registration.id,
     identifier: registration.identifier,
-    createdDate: registration.createdDate,
-    modifiedDate: registration.modifiedDate,
-    publishedDate: registration.publishedDate,
-    status: registration.status,
-    entityDescription: {
-      mainTitle: registration.entityDescription?.mainTitle ?? '',
-      abstract: registration.entityDescription?.abstract ?? '',
-      description: registration.entityDescription?.description ?? '',
-      publicationDate: registration.entityDescription?.publicationDate,
-      contributorsPreview: registration.entityDescription?.contributors ?? [],
-      contributorsCount: (registration.entityDescription?.contributors ?? []).length,
-      reference: {
-        publicationInstance: {
-          type: registration.entityDescription?.reference?.publicationInstance?.type,
-        },
-        publicationContext: {
-          publisher,
-          series,
-        },
+    recordMetadata: {
+      createdDate: registration.createdDate,
+      modifiedDate: registration.modifiedDate,
+      publishedDate: registration.publishedDate,
+      status: registration.status,
+    },
+    mainTitle: registration.entityDescription?.mainTitle ?? '',
+    contributorsCount: contributors.length,
+    abstract: registration.entityDescription?.abstract ?? '',
+    description: registration.entityDescription?.description ?? '',
+    publicationDate: registration.entityDescription?.publicationDate,
+    publishingDetails: {
+      id: journalId,
+      type: publishingContextType as PublicationType,
```

Comment:
Der denne defineres kan den være undefined også. Kanskje flytte typenspesifisering dit i stedet?
```suggestion
      type: publishingContextType,
```

```
  const publishingContextType =
    registration.entityDescription?.reference?.publicationContext &&
    'type' in registration.entityDescription.reference.publicationContext
      ? registration.entityDescription.reference.publicationContext.type as PublicationType
      : undefined;
```